src/bionics.h

```diff
@@ -80,6 +80,8 @@ struct bionic_data {
     float weight_capacity_modifier;
     /**Bonus to weight capacity*/
     units::mass weight_capacity_bonus;
+    /**Stat bonus*/
```

Comment:
Seriously: why this comment? It literally repeats the name of the member, it adds *no* information at all.